### PR TITLE
clouway-incubator/I/O: read data from standard input and save it in file

### PR DESCRIPTION
**Pull request by qnislavnachev**
Here we are implemented functionality which read from standard input data (String, Integer, Float and Char) and save it in file. When we read only symbol ```.```` (point) program terminate. In the beginning of program the client has to say filename in which we will save input data. 

This covers first and second tasks.

### CODE REVIEW

**Comment by qnislavnachev** [comment]
Exercise 1 and 2 from "Input-Output" done and ready for review!

**Comment by EmilGeorgiev** [comment]
```Exercise 1 and 2 from "Input-Output" done and ready for review!``` is bad name for pull request. Also you should describe with one or two sentences what exactly represent this pull request

File: .idea/uiDesigner.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (EmilGeorgiev) [inline .idea/uiDesigner.xml]: do not forget to add all unneccesary files in .gitingnore

File: src/task2/Demo.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+public class Demo {
+    public static void main(String[] args) throws IOException {
+        Scanner scan = new Scanner(System.in);
+        FileWriter writer = new FileWriter(new File("MyFile.md"));
+        System.out.println("Enter some text: ");
+        String text;
+        do {
+            text = scan.nextLine();
+        }while(!text.contains("."));
```
Comment (EmilGeorgiev) [inline src/task2/Demo.java]:
```text
Четенето спира при въвеждане на ред съдържащ само една точка ".".
```

We should terminate program if row contains only one character ".". And we should save input data in file every time when read line. Here we will save only one row in file

File: src/task1/Demo.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package task1;
+
+import java.util.Scanner;
+
+public class Demo {
+    public static void main(String[] args) {
+        ConsoleReader reader = new ConsoleReader();
+        Scanner scan = new Scanner(System.in);
```
Comment (EmilGeorgiev) [inline src/task1/Demo.java]: this Scanner should be in ConsoleReader. Aim of class ConsoleReader is to read input data from console.  Whit your implementation, we can use every class to read from console. For example:  

```java
public class Person {
    public String name(Staring name) {
         return name
    }

   public Integer age(Integer age) {
         return age
    }
}

public class DemoConsoleReader {
    public static void main(String[] args) {
         Person person = new Person()
         Scanner scan = new Scanner(System.in)

         person.name(scan.nextLine())
         person.age(scan.nextInt())
   }
}
``` 

This class person is not many different from your ConsoleReader.

File: src/task1/WrongValueException.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package task1;
+
+public class WrongValueException extends RuntimeException{
```
Comment (EmilGeorgiev) [inline src/task1/WrongValueException.java]: Words as ```wrong```, ```incorect```, ```validator```, ```valid``` are to common. Rename to ```InputDataException``` and when throw this exception write descriptive message and pass them as parameter of its constructor.

**Comment by EmilGeorgiev** [comment]
LGTM

**Pull request merged**